### PR TITLE
Tornado port

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

File: cesium_app/handlers/task.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# TODO pass in user somehow
+# TODO show error notification for failures
+@app.route("/task_complete", methods=['POST'])
+def task_complete():
```
Comment (bnaul) [inline cesium_app/handlers/task.py]: This file can be deleted, right?

**Comment by bnaul** [comment]
What about the `task_server`? If that's sticking around, we could prune the logic that's no longer used. Otherwise 👍  pending tests!

**Comment by acrellin** [comment]
Which test got removed?

**Comment by bnaul** [comment]
What do you mean?

**Comment by bnaul** [comment]
Oh did the total # change? Could have been a merge conflict mistake or something...

**Comment by acrellin** [comment]
There were 21 tests running on my PR that just got merged, and 20 here (from the Travis logs).

**Comment by acrellin** [comment]
Yeah, exactly.

**Comment by bnaul** [comment]
Ah it's because `check_model_param_types` was removed.

**Comment by acrellin** [comment]
Ah, okay. Thanks for clearing that up.

**Pull request merged**